gradle/testing/randomization/policies/solr-tests.policy

```diff
@@ -109,6 +109,8 @@ grant {
   permission java.lang.RuntimePermission "writeFileDescriptor";
   // needed by hadoop http
   permission java.lang.RuntimePermission "getProtectionDomain";
+  // SolrProcessMgr to list processes
+  permission java.lang.RuntimePermission "manageProcess";
```

Comment:
Note that we need to grant "manageProcess" permission only for tests, since tests run under the security manager and list and spawn processes. But the only current use of SolrProcessManager is from StatusTool which runs in SolrCLI which is not invoked with security manager.

If we want Solr's runtime to use SolrProcessManager in the future, we'd need to add this permission to `security.policy` as well. Not doing that since it would grant a potential attacker the right to not only list processes started by others users, but also terminate them.

Reply:
Doesn't the security manager go away when we update to java 21 and lucene 10?

Reply:
You're right. But since this targets 9.x, we still need it. When someone rips out security manager related stuff in main before the 10.0 release, this will be gone of course. But not right now..

Reply:
I would be fine with this as 10 if that makes your life easier..   kind of like the basic auth for CLI being 10x only...   I am assuming 10x is only months away!

Reply:
No big deal, it will anyway land in main first. And if security manager is removed in main branch before this is merged, we'll of course have to take action..